optimum/intel/neural_compressor/utils.py

```diff
@@ -27,6 +28,9 @@
 WEIGHTS_NAME = "pytorch_model.bin"
 TRAINING_ARGS_NAME = "training_args.bin"
 
+parsed_torch_version_base = version.parse(version.parse(torch.__version__).base_version)
+is_torch_less_than_1_13 = parsed_torch_version_base < version.parse("1.13.0")
```

Comment:
I have seen similar variables / functions for @younesbelkada PR. I think it would make more sense to make a file `optimum/utils/version_utils.py` to avoid duplicates with different names, and allow easier maitainability.

Reply:
Yes, happy to take care of that in my PR, or open a follow up PR for that, your call!

Reply:
In your PR it's perfect, thanks @younesbelkada !